src/app/hooks/useLayer.ts

```diff
@@ -0,0 +1,12 @@
+import { useParams } from 'react-router-dom'
+import { Layer } from '../../config'
+
+export const useLayer = () => {
```

Comment:
not used atm? what would be a use case? In views we will use `const { layer } = useParams()` probably

Reply:
Good point, redundant.